packages/ripple-binary-codec/test/tx-encode-decode.test.js

```diff
@@ -22,7 +26,7 @@ describe('encoding and decoding tx_json', function () {
   test('can encode tx_json without Amount or Fee', function () {
     const encoded = encode(tx_json)
     const decoded = decode(encoded)
-    expect(tx_json).toEqual(decoded)
```

Comment:
Why was this changed?

Reply:
When running it was saying that `chai has no attribute toEqual`, so I think the syntax may have changed to `.deep.equal`.

Reply:
This isn't chai - this is jest. Don't worry about changing the whole structure of the tests in this PR.

Reply:
That makes sense